src/Directory.Build.targets

```diff
@@ -15,4 +15,19 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
   </ItemGroup>
 
+  <Target Name="UseExplicitPackageVersions" BeforeTargets="GenerateNuspec">
```

Comment:
```suggestion
  <!--Workaround for https://github.com/NuGet/Home/issues/5556 and https://github.com/NuGet/Home/issues/5525-->
  <Target Name="UseExplicitPackageVersions" BeforeTargets="GenerateNuspec">
```

Reply:
Without an explanatory comment, an ordinary developer (i.e not MSBuild guru) is completely impossible to understand what is happening here.